pkg/cri/sbserver/podsandbox/sandbox_stop.go

```diff
@@ -90,6 +91,11 @@ func (c *Controller) stopSandboxContainer(ctx context.Context, sandbox sandboxst
 		go func() {
 			defer close(stopCh)
 			exitStatus, exitedAt, err := c.waitSandboxExit(exitCtx, id, exitCh)
+			sandbox.Status.Update(func(status sandboxstore.Status) (sandboxstore.Status, error) {
+				status.ExitedAt = exitedAt
```

Comment:
I don't understand why we update status here even if we got err here?

Reply:
When err is not nil, `exitStatus=unknownExitCode(255)`、`exitedAt=time.Now()`；This is logging its exit status, which I think should be logged regardless of success or failure; what do you think could be the problem here?